pkg/cluster/task/check.go

```diff
@@ -188,6 +188,19 @@ func (c *CheckSys) runFIO(ctx context.Context) (outRR []byte, outRW []byte, outL
 	testWd := filepath.Join(checkDir, "tiup-fio-test")
 	fioBin := filepath.Join(CheckToolsPathDir, "bin", "fio")
 
+	notExistDir := testWd
+	for {
+		parent := filepath.Dir(notExistDir)
```

Comment:
Is not the parent `checkDir`? Can not we just use `checkDir`?

Reply:
checkDir is the `data/pd` or `data/tikv` directory in my case, but typically I need to remove `data`.